target_postgres/postgres.py

```diff
@@ -543,20 +542,48 @@ def get_update_sql(self, target_table_name, temp_table_name, key_properties, sub
                         insert_distinct_on=insert_distinct_on,
                         insert_distinct_order_by=insert_distinct_order_by)
 
+    def serialized_value(self, field, value, default_fields, datetime_fields):
```

Comment:
@AlexanderMann I don't think we want to combine defaulting and serialization (eg formatting dates). Those seem like distinct things to me. I think creating true functionality around serialization, which in this case is a CSV, may make sense. As in, call a serialize function per row that returns a CSV row, and serialize each field as needed.